components/Tabs/src/private/MDCItemBar.m

```diff
@@ -1,5 +1,6 @@
 /*
```

Comment:
I ran this file through clang-format, but my intent was that the only significant change was from willSelectItem to shouldSelectItem.